brake/backends/cachebe.py

```diff
@@ -71,7 +73,7 @@ def limit(self, func_name, request,
             if ':ip:' in counter:
                 ratelimited_by = 'ip'
 
-            if counters[counter] > count:
+            if counters[counter][0] > count:
```

Comment:
Before it was a scalar, now it's a tuple.

Not super worried about it, but this needs some careful testing.